core/value_function.cc

```diff
@@ -49,6 +49,11 @@ toValueInfo(std::shared_ptr<const ValueDescription> desc)
                 info.desc = field;
                 std::tie(info.info, info.fromInput, info.toOutput)
                     = toValueInfo(field.description);
+
+                if (!info.info)
+                    throw HttpReturnException
+                            (500, "No info for field " + info.fieldName.toUtf8String());
```

Comment:
Let's add the field.description to the exception so that if this exception triggers we know which type we need to implement.  Also, this check should probably be in toValueInfo and we can document that it won't return null pointers.